libs/dh/admin/feature-user-management/src/lib/user-management-tabs/dh-users-tab.component.html

```diff
@@ -0,0 +1,44 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'admin.userManagement.tabs.users'">
+  <watt-card>
+    <watt-card-title>
+      <ng-container
+        *ngTemplateOutlet="
+          usersHeadline;
+          context: {
+            $implicit: numberOfUsers$ | push
+          }
+        "
+      ></ng-container>
+    </watt-card-title>
+  </watt-card>
+</ng-container>
+
+<ng-template #usersHeadline let-numberOfUsers>
+  <ng-container
+    *transloco="let t; read: 'admin.userManagement.tabs.users.usersHeadline'"
+  >
+    <h4 *ngIf="numberOfUsers === 1">
+      {{ t("singular", { numberOfUsers: numberOfUsers }) }}
```

Comment:
Suggestion: use https://2ality.com/2019/12/intl-pluralrules.html to handle singular and plural